### PR TITLE
Dmp 1067.audit request audio

### PR DESCRIPTION
**Pull request by HMCTSDeveloper**
https://tools.hmcts.net/jira/browse/DMP-1067

When a media request is created, create an audit entry for the "Request Audio" activity and the relevant user ID.

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/darts/common/exception/AuditApiError.java [hunk 1]
```diff
@@ -17,6 +17,11 @@ public enum AuditApiError implements DartsApiError {
         "101",
         HttpStatus.BAD_REQUEST,
         "When using date filters, both must be provided."
+    ),
+    NO_HEARING_OR_USER_FOUND_WHEN_ADDING_AUDIO_AUDIT(
+        "101",
```
Comment (cmagowan-hmcts) [inline src/main/java/uk/gov/hmcts/darts/common/exception/AuditApiError.java:22]: code should be different

File: src/main/java/uk/gov/hmcts/darts/common/exception/AuditApiError.java [hunk 2]
```diff
@@ -17,6 +17,11 @@ public enum AuditApiError implements DartsApiError {
...
+        HttpStatus.BAD_REQUEST,
+        "No hearing or user found when auditing add audio."
```
Comment (cmagowan-hmcts) [inline src/main/java/uk/gov/hmcts/darts/common/exception/AuditApiError.java:24]: should `add audio` either be one word or surrounded by quotes?

File: src/main/java/uk/gov/hmcts/darts/common/exception/AuditApiError.java (hunk 1 above)
Comment (HMCTSDeveloper) [reply src/main/java/uk/gov/hmcts/darts/common/exception/AuditApiError.java:22]: ACCEPT

File: src/main/java/uk/gov/hmcts/darts/common/exception/AuditApiError.java (hunk 2 above)
Comment (HMCTSDeveloper) [reply src/main/java/uk/gov/hmcts/darts/common/exception/AuditApiError.java:24]: Ill change it to No hearing or user found when auditing 'add audio' request.

**Review by cmagowan-hmcts — APPROVED** [review]

**Pull request merged**